src/main/java/com/woowacourse/matzip/application/RestaurantService.java

```diff
@@ -30,9 +32,23 @@ public List<RestaurantTitleResponse> findByCampusIdOrderByIdDesc(final Long camp
                 .collect(Collectors.toList());
     }
 
-    private RestaurantTitleResponse toResponse(Restaurant restaurant) {
-        double rating = reviewRepository.findAverageRatingByRestaurantId(restaurant.getId())
+    public List<RestaurantTitleResponse> findRandomByCampusId(final Long campusId, final int size) {
+        return restaurantRepository.findRandomByCampusId(campusId, size)
+                .stream()
+                .map(this::toResponse)
+                .collect(Collectors.toList());
+    }
+
+    private RestaurantTitleResponse toResponse(final Restaurant restaurant) {
```

Comment:
```suggestion
    private RestaurantTitleResponse toRestaurantTitleResponseWithRating(final Restaurant restaurant) {
```
조금 더 구체화하는건 어떻게 생각하세요?

Reply:
`toRestaurantTitleResponse` 부분까지는 공감하는 내용입니다! 다만 `WithRating` 부분은 `rating`을 파라미터로 받는 형태가 아니고 메서드 내에서 구하는데다가 어차피 `RestaurantTitleResponse`는 무조건 `rating`이 들어가야 한다는 점을 생각해 봤을 때 과한 네이밍이라고 생각해요(메서드 내부에서 `rating`을 구해서 집어넣어 주는걸 외부에서 알 필요가 있을까 싶네요)
그래서 일단은 `toRestaurantTitleResponse` 정도로만 수정할게요 혹시 다른 의견 있으면 계속 남겨주세요!